pages/guides/installation.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Getting Started
+description: How to install and set up Chakra UI in your project
+tags: ['principles']
+---
+
+## Watch the Egghead Course
+
+Hello and welcome to Chakra UI 😄. If you're new here, watch our free course
+below.
+
+<EggheadCourseBanner href='https://egghead.io/courses/build-a-modern-user-interface-with-chakra-ui-fac68106' />
+
+## Framework Guide
+
+Whichever framework or tool you're using, we've put together step-by-step guides
+to get you up and running with Chakra UI.
+
+<FrameworkLinks />
+
+## Use our Create-React-App Templates
+
+> - For Create React App, you need to set this up in `index.js` or `index.tsx`
```

Comment:
We're showing a blockquote that says that you need to set "this" up in a certain way for CRA. But, since we moved the instructions out of this file "this" refers to nothing now.

Reply:
Also, I think we should merge the `/guides/integrations/with-cra` and `/guides/getting-started/cra-guide` pages.

Reply:
Right good Idea!

Reply:
I'll put both under getting-started, what do you think ?